code/src/components/NewTask.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import uniqid from "uniqid";
+
```

Comment:
Nice that you follow the guidelines and put a empty line beween different imports :-)